python/hail/docs/utils/index.rst

```diff
@@ -3,6 +3,8 @@ utils
 
 .. currentmodule:: hail.utils
 
+.. autoclass:: Struct
```

Comment:
Is this the right place for `Struct`?

Reply:
hail.utils? I think so. Where else should it be?